docs/quicktips/slicer-upload.md

```diff
@@ -9,27 +9,26 @@ description: >-
 
 # {{ page.title }}
 {{ page.description }}  
-Moonraker has compatibility with Octoprint's API. To enable octoprint api compatibility,
-add the following to your `moonraker.conf` and restart Moonraker.
+Moonraker is compatible with Octoprint's API. To enable Octoprint API compatibility, add the following to your `moonraker.conf` and restart Moonraker.
 
 ```
 [octoprint_compat]
 ```
 
 
-### PrusaSlicer/SuperSlicer (prior to v2.3.0)
+### PrusaSlicer / SuperSlicer (Mainsail v2.3.0 and up)
```

Comment:
Nope, not Mainsail. It's about the slicers. They changed at some point how to connect to the printers.